client/src/locales/en-US.json

```
@@ -523,12 +523,17 @@
         },
         "sort": {
             "sortByName": "Name",
+            "sortByRole": "Role",
             "sortBySize": "Size",
             "sortByLastUpdated": "Last updated"
         },
         "getLinkError": "Failed to get a link to the workspace",
         "linkCopiedToClipboard": "Workspace link has been copied to clipboard.",
-        "linkNotCopiedToClipboard": "Failed to copy the link. Your browser or device does not seem to support copy/paste."
+        "linkNotCopiedToClipboard": "Failed to copy the link. Your browser or device does not seem to support copy/paste.",
+        "filter": {
+            "title": "Filter",
+            "roles": "Roles"
```

To be consistent with the Profile filter maybe should be displayed in singular.
```suggestion
            "roles": "Role"
```